tree/tree/inc/TTree.h

```diff
@@ -98,6 +100,7 @@ class TTree : public TNamed, public TAttLine, public TAttFill, public TAttMarker
    Int_t          fDebug;                 ///<! Debug level
    Long64_t       fDebugMin;              ///<! First entry number to debug
    Long64_t       fDebugMax;              ///<! Last entry number to debug
+   ROOT::TIOFeatures fIOFeatures{0};      ///<! IO features to define for newly-written baskets and branches.
```

Comment:
May add a using TIOFeatures = ROOT::TIOFeatures; above to short and align this line.